js/core/css/caniuse.css

```diff
@@ -24,13 +28,24 @@
 
 /* a browser version */
 .caniuse-cell {
-  position: relative;
-  min-width: 3cm;
-  padding: 4px;
-  background: #eee; /* default, for unknown support */
-  margin: 1px;
-  margin-left: 0;
+  color: rgba(0, 0, 0, 0.8);
+  background: #eee;
+  border: none;
+  font-size: 90%;
+  height: 0.8cm;
+  margin-right: 1px;
   margin-top: 0;
+  min-width: 3cm;
+  overflow: visible;
+  position: relative;
+  line-height: 0.79cm;
+  display: flex;
```

Comment:
If the parent has display flex, does this need to be display flex?

Reply:
Yes, I think so. As we need to center content in the current element (`<button>`)

With an aim for pixel perfect CSS, have a look at this little video clip: https://streamable.com/lye93

Reply:
Thanks for clarifying!